src/main/java/Duke.java

```diff
@@ -1,10 +1,50 @@
+import java.util.Scanner;
+
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        //Generates a greeting
+        greeting();
+        request();
     }
-}
+
+    public static void request(){
+        Scanner in = new Scanner(System.in);
+        Task[] list = new Task[100];
```

Comment:
instead of naming it list, name it "taskList" so that when you create multiple lists, it is clear what each list is for.